components/org.wso2.carbon.identity.verification.onfido.api/org.wso2.carbon.identity.verification.onfido.api.common/src/main/java/org/wso2/carbon/identity/verification/onfido/api/common/ContextLoader.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.verification.onfido.api.common;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+import org.wso2.carbon.utils.multitenancy.MultitenantConstants;
+
+import static org.wso2.carbon.identity.verification.onfido.api.common.Constants.TENANT_NAME_FROM_CONTEXT;
+
+/**
+ * Load information from context.
+ */
+public class ContextLoader {
+
+    /**
+     * Retrieves loaded tenant domain from carbon context.
+     *
+     * @return tenant domain of the request is being served.
+     */
+    public static String getTenantDomainFromContext() {
+
+        String tenantDomain = MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
+        if (IdentityUtil.threadLocalProperties.get().get(TENANT_NAME_FROM_CONTEXT) != null) {
+            tenantDomain = (String) IdentityUtil.threadLocalProperties.get().get(TENANT_NAME_FROM_CONTEXT);
+        }
+        return tenantDomain;
```

Comment:
```suggestion
        if (IdentityUtil.threadLocalProperties.get().get(TENANT_NAME_FROM_CONTEXT) != null) {
            return (String) IdentityUtil.threadLocalProperties.get().get(TENANT_NAME_FROM_CONTEXT);
        }
        return MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
```

We can remove the redundant variable initialization here.

Reply:
Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/12/commits/6f73cff0b813bfcfffbd6cfbe9d730df1f5fe3ed